packages/react-components/src/templates/__tests__/ResearchOutputPage.test.tsx

```diff
@@ -1,19 +1,42 @@
-import React from 'react';
+import React, { ComponentProps } from 'react';
 import { render } from '@testing-library/react';
 
 import ResearchOutputPage from '../ResearchOutputPage';
 
+const props: ComponentProps<typeof ResearchOutputPage> = {
+  libraryHref: '#',
+  profileHref: '#',
+  title: 'title',
+  type: 'proposal',
+  created: '2020-06-25T15:00:47.920Z',
+  text: 'content',
+};
 it('renders a proposal title and content', () => {
   const { getByText } = render(
     <ResearchOutputPage
-      id="uuid"
-      title="title"
+      {...props}
       type="proposal"
-      created="2020-06-25T15:00:47.920Z"
+      title="title"
       text="content"
     />,
   );
   expect(getByText(/proposal/i)).toBeVisible();
-  expect(getByText(/title/i, { selector: 'h1' })).toBeVisible();
+  expect(getByText(/title/i, { selector: 'h3' })).toBeVisible();
```

Comment:
This shouldn't pass? If this still passes with `h3` then the text `title` you used is probably too generic because it clearly selects something else from the generated text

Reply:
It didn't pass 😅